src/widgets/NumericTextCtrl.h

```diff
@@ -57,14 +58,16 @@ class AUDACITY_DLL_API NumericTextCtrl final
       Options &InvalidValue (bool has, double v = -1.0)
          { hasInvalidValue = has, invalidValue = v; return *this; }
       // use a custom format not in the tables:
-      Options &Format (const FormatStrings &f)
-         { format = f; return *this; }
+      Options& FormatSymbol(const NumericFormatSymbol& f)
+         { formatSymbol = f; return *this; }
+      Options& CustomFormat(const TranslatableString& f)
+         { customFormat = f; return *this; }
```

Comment:
A minor point.  These two are not made mutually exclusive and custom will quietly override the symbol in case someone give both.  But who cares.